ovos_phal_plugin_hotkeys/boards.py

```diff
@@ -0,0 +1,13 @@
+WM8960  = {"key_down": {"mycroft.mic.listen": 143}}
+
+SJ201 = {
```

Comment:
i would consider defining these as json files in a XDG path, defaults could be created on the package install (setup.py) or plugin validator? this also allows images/package_managers/installer to create a .json file on install time

we could then drop boards.py ?


can be done in a follow up PR, just throwing the idea out there